en_US/data-integration/data-bridge-iotdb.md

```diff
@@ -101,30 +123,7 @@ This section demonstrates how to create a rule in EMQX to process messages from
      *
    FROM
      "root/#"
-   
-   ```
-   If you need to specify your own rule, you need to include the required contextual information in the MQTT message in the `SELECT` part of the rule. For example, the client is sending a message with the payload in JSON format as follows: 
-   ```json
-   {
-     "measurement": "temp",
-     "data_type": "FLOAT",
-     "value": "32.67",
-     "device_id": "root.sg27" // optional
-   }
-   ```
-   You can use the following rule to present the fields `measurement`, `data_type` and `value`.
-   ```sql
-   SELECT
-     payload.measurement, payload.data_type, payload.value, clientid as payload.device_id
-   FROM
-     "root/#"
-   ```
-   If the payload is structured differently, you can use the rule to rewrite its structure like the following:
-   ```sql
-   SELECT
-     payload.measurement, payload.dtype as payload.data_type, payload.val as payload.value
-   FROM
-     "root/#"
+
    ```
 
 5. Click the **Add Action** button, and select **Forwarding with Data Bridge** from the dropdown list. With this action, EMQX sends the data processed by the rule to the data bridge.
```

Comment:
```suggestion
5. Click the **+ Add Action** button to define an action that will be triggered by the rule. Select `Apache IoTDB` from the **Type of Action** dropdown list so that EMQX will send the data processed by the rule to Apache IoTDB. 
  Keep the **Action** dropdown box with the value `Create Action`. Or, you also can select a Apache IoTDB Sink previously created. In this demonstration, you create a new Sink and add it to the rule.
```